tools/bin/hawqpylib/hawqlib.py

```diff
@@ -203,6 +203,13 @@ def check_hostname_equal(remote_host, user = ""):
     cmd = "hostname"
     result_local, local_hostname, stderr_remote  = local_ssh_output(cmd)
     result_remote, remote_hostname, stderr_remote = remote_ssh_output(cmd, remote_host, user)
+    if result_remote != 0:
+        print stderr_remote
+        print "Execute remote command failed on %s." % remote_host
```

Comment:
For Q1, added command line and error code in the error message.
For Q2, there are enough error information inside stderr_remote. User can read and know what's happening.